packages/language-server/src/server.ts

```diff
@@ -88,6 +88,10 @@ connection.onInitialize(async (params) => {
 			},
 		})
 		service.project.on('documentErrored', async ({ errors, uri, version }) => {
+			if (!uri.startsWith('file://')) {
```

Comment:
Is it a good idea to explicitly check for `file` rather than filter out `archive` explicitly? Are we sure this won't break stuff like remote ssh, live share, or others?

Reply:
Both the intellij built in lsp client and lsp4intellij check explicitly for file://. If there's a case where vscode accepts other urls I can check for archive:// instead

Reply:
Afaik VS Code uses the `untitled:` URI scheme for new files that are not saved on disk